src/applications/mhv-medical-records/api/MrApi.js

```diff
@@ -67,6 +69,25 @@ export const getLabsAndTests = async () => {
     headers,
   });
 };
+export const getAcceleratedLabsAndTests = async timeFrame => {
+  const from = `from=${timeFrame}`;
```

Comment:
confirm this will work with the [labs and tests v2 controller](https://github.com/department-of-veterans-affairs/vets-api/blob/4c1bdedbad5dbb9f50cae90b275b781aa1f41e65/modules/my_health/app/controllers/my_health/v2/labs_and_tests_controller.rb#L12). might need to be `startDate` and `endDate`?